lib/arkana.rb

```diff
@@ -43,7 +44,14 @@ def self.run(arguments)
       config: config,
       salt: salt,
     )
-    SwiftCodeGenerator.generate(
+
+    generator = case config.current_lang.downcase
+      when "swift" then SwiftCodeGenerator
+      when "kotlin" then KotlinCodeGenerator
+      else UI.crash("Unknown output lang selected: #{config.current_lang.downcase}")
```

Comment:
Do we need to change `current_lang` to lower case when it is displayed in the error message? 

It's a small detail, but IMO it would be slightly easier for the user to identify their mistake if we just printed the argument as they provided it, without changing the case of the letters.

```suggestion
      else UI.crash("Unknown output lang selected: #{config.current_lang}")
```